### DISCUSSION TITLE
impossibile accedere ad HA da altri pc e da App

### DISCUSSION THREAD

**Discussion by MapoCaos** [indomus/forum#2813] [HUB: Home Assistant]
Ciao a tutti,
Ho un problema di accesso ad HA da altri pc e dall'app, riesco solo ad accedere dal mini pc e solo mediante indirizzo ip: 192.168.178.xxx.:8123 e non da http://homeassistant.local:8123/. Ho appena fatto l'installazione di HA supervised (ho optato per questa versione xk non sono pratico di linux e non ho tanto tempo da dedicarci.. poi volevo installare anche Pi- hole e quindi la versione HA OS non era adatta credo...) su un mini-pc testato da voi e acquistato appositamente. Il pc ha 512gb di ssd e quindi ho deciso di installare in Dual-boot Debian 12 e win11 pro che era già presente, nel caso mi dovessere servire in futuro. Ho lasciato metà disco ad entrambi gli OS.
Ho seguito la guida e l'unica cosa che non sono riuscito a fare e la connessione tramite VNC (ho seguito anche diverse guide ma nulla), mentre quella tramite SSH sono riuscito.
Ho un fritzbox 7530 con rete fibra Tim e utilizzo nextdns direttamente dal router per togliere la pubblicità. Ho provato a disattivare nextdns ma non cambia nulla. Ho provato anche da router a impostare un ip fisso dandogli sempre lo stesso ip e cambiando il numero finale con un elevato per non riasciare di avere problemi con altro. Avevo provato a far girare HA su un vecchio pc windows ed ero riuscito a collagrmi da app senza problemi adesso ho testato anche questo e non funziona più. Al momento sono bloccato e non so come fare.. può essere qualche blocco del router? Non sono pratico di linux (questa è la prima volta) Sapreste darmi una mano?
Grazie mille
EDIT: " ho provato con il vecchio pc Win10 su cui ho HA e funziona e riesco ad eccedere da altri pc. A questo punto ho smarcato l'ipotesi del problema al router..

**Comment by trebestie** [comment]
Magari ho frainteso il problema, ma c'è qualcosa che risolve homeassistant.local?
Se su un pc da cui ti colleghi fai un banale 'ping homessistant.local', cosa ti dice?

**Reply by KC-inDomus** [reply]
se non funziona, molto semplicemente e probabilmente non ha mdns attivo sul router

**Reply by MapoCaos** [reply]
Ciao grazie mille per le risposte.
Alla fine ho provato a reinstallere Debian 12 su tutto il pc togliendo windows e rifacendo l'installazione e adesso funziona tutto. Probabilmente a questo punto ho fatto qualche errore io durante l'installazione di HA.
mdns non l'ho ancora attivato xk non ho capito come si fa..

Grazie ancora
